### DISCUSSION TITLE
【应用投递】基于现行 fcitx5 框架下的 fcitx5-rime 98五笔套件

### DISCUSSION THREAD

**Discussion by yanhuacuo** [linuxdeepin/developer-center#4684] [General | 常规讨论]
fcitx5-rime 基于 librime 的长足进步，早已是硕果累累，今非夕比。

尤其是 lua 插件的支持，使得 fcitx5-rime 在功能的扩展性上，远胜 fcitx4 及市面上一切商业输入法应用：

表现在：

- 可以显示汉字五笔字根的拆分
- 可以支持万年历转换
- 可以支持数制转换
- 可以支持翻简转换
- 可以随时转换字符集
- 支持音形互通的反查

在实现上述功能的基础上，再辅以：

- 全面可视化的功能控制
- 百万级别码表管理器
- 等效实现小狼毫、鼠须管等全面功能

https://github.com/yanhuacuo/rime98wb

这个应用，申请入驻应用商店。
开源协议：GPLV3.0

请开发人员测试一下，盼复。

**Comment by BLumia** [comment]
我想确认一下，您这边是更侧重于希望软件包进入系统仓库还是进入应用商店并在商店陈列展示你的应用？

如果是更侧重进入社区版系统仓库的话，可以直接参与软件包的递交与维护，可以参见 [deepin-community协作指南](https://wiki.deepin.org/deepin-community%E5%8D%8F%E4%BD%9C%E6%8C%87%E5%8D%97) 了解软件包的递交与维护流程。（我个人更推荐走这个方案）

如果是更侧重进入商店的话，是在 chinauos 网站上进行应用软件的递交。

**Reply by yanhuacuo** [reply]
好的，我抽时间学习一下。
